wit/network.wit

```diff
@@ -13,11 +13,77 @@ default interface network {
 	drop-network: func(this: network)
 
 
-
+	/// Error codes.
+	/// 
+	/// In theory, every API can return any error code.
+	/// In practice, API's typically only return the errors documented per API
+	/// combined with a couple of errors that are always possible:
+	/// - `unknown`
+	/// - `access-denied`
+	/// - `not-supported`
+	/// - `resource-limit-reached`
+	/// - `out-of-memory`
+	/// 
+	/// See each individual API for what the POSIX equivalents are. They sometimes differ per API.
 	enum error {
```

Comment:
For consistency with wasi-filesystem as of https://github.com/WebAssembly/wasi-filesystem/pull/102, could you rename this to `error-code` (or we can change wasi-filesystem; I don't have a strong preference other than wanting them to be consistent).

Reply:
Don't have a preference either. I've renamed it in: 0232ce65349ed027fa33a56663af9cd3de994db0